Tests/Functional/ConfigurationTest.php

```diff
@@ -18,18 +18,23 @@ class ConfigurationTest extends WebTestCase
 {
     public function testDisabledTemplating()
     {
-        $this->createClient(['test_case' => 'Templating']);
+        $kernel = self::bootKernel(['test_case' => 'Templating']);
+        $container = $kernel->getContainer();
+
+        $this->assertFalse($container->has('fos_rest.templating'));
```

Comment:
added some assertions to tests that didn't have them so that they are not reported as risky by PHPUnit